src/components/overview/components/OfferingCard/OfferingCard.tsx

```diff
@@ -75,6 +76,7 @@ function OfferingCard(props: OfferingCardProps) {
 
   let offeringCardTitle: string | undefined;
   let offeringCardLabel: string = 'Managed service';
+  let offeringCardShowSupportLevelBadge: boolean = false;
```

Comment:
Nitpicking comment but worth bringing to your attention IMHO
At first I didn't understand why you set it to false when you later on set it to true, but then I realized it was just for the case of `MIGRATION` type of OfferingCard (`OfferingType` prop)

Consider defining it as a constant and check the `OfferingType` on the definition. It seems a little confusing for me as it is super easy logic to understand, temporary and doesn't necessarily related DIRECTLY to the OfferingType of the OfferingCard (as per say it's title or description)
Something like this: 
```suggestion
  const shouldShowSupportLevelBadge = offeringType === 'MIGRATION' ?? false;
```

Reply:
@LiorKGOW I've added this property because maybe in the future we want to add the PreviewBadge to other cards.